meshery.sh

```diff
@@ -35,7 +35,7 @@ main() {
 	kubectl config view --minify --flatten > ~/minified_config
 	mv ~/minified_config ~/.kube/config
 
-  curl -L https://git.io/meshery | PLATFORM=$PLATFORM bash -
+  	curl -L https://git.io/meshery | sudo PLATFORM=$PLATFORM bash - &
```

Comment:
Any reason for running this command in the background?

Reply:
When I tested on the self-hosted runner, it will be pending always if not running it in the background. But I don't know why this doesn't happen on github runner.

Reply:
Hmm okay, it's a minor thing let's just keep a note of this behaviour and come back to it later.

Reply:
Yes, it's a good reminder.